dev-python/kaptan/kaptan-0.5.8.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Configuration manager in your pocket"
```

Comment:
"Configuration manager in your pocket" could be anything. Could you elaborate on this DESCRIPTION, please?

Reply:
I'll blame that on the shitty github description..... :grinning:

Perhaps something more fitting would be "Multiple config format parser" but that's verging on gibberish too .. ideas?!  :anguished: